doc/sphinx/source/getting_started/running.rst

```diff
@@ -54,3 +54,54 @@ Available diagnostics and metrics
 
 See Section :doc:`Recipes <../recipes/index>` for a description of all
 available recipes.
+
+
+Some pro tips on running
+========================
+
+Re-running diagnostics
+----------------------
+If a diagnostic fails, you will get the message
+
+.. code:: bash
+
+   INFO    To re-run this diagnostic script, run:
+
+Ifg you run the command in the stdout you will be able to re-run the
+diagnostic without having to re-run the whole preprocessor. If you add the ``-f``
+argument that will force an overwrite, and it will deletes not just the failed diagnostic,
+but the contents of the entire ``plots`` directory - this is risky but useful when needing to
+redo the whole work. Adding ``-i`` or ``--ignore-existing`` will not delete any existing files,
+and it will re-do the work related to the failed diagnostic only.
+
+
+Enter interactive mode with iPython
+-----------------------------------
+Sometimes it is useful to enter an interactive session to have a look what's going on.
+Insert a single line in the code where you want to enter IPython:
+``import IPython; IPython.embed()``
+
+This is a useful functionality because it allows the user to `fix` things on-the-fly and after
+quitting the Ipython console, code execution continues as per normal.
+
+
```

Comment:
Here's another pro-tip:

```suggestion

Use multiple config-user.yml files
----------------------------------

The user profiles the configuration yaml file at run time. It's possible to 
have several configurations files. For instance, it may be practical to have one 
config file for debugging runs and another for production runs.

```